lib/cob_index/default_config.rb

```diff
@@ -0,0 +1,29 @@
+module CobIndex
+  module  DefaultConfig
+    # Wrapper for our default indexer settings so that we can share them
+    # across commands.
+    def self.indexer_settings
+      solr_url = if File.exist? "config/blacklight.yml"
+                      solr_config = YAML.load_file("config/blacklight.yml")[(ENV["RAILS_ENV"] || "development")]
+                      ERB.new(solr_config["url"]).result
+                    else
+                      ENV["SOLR_URL"] || "http://localhost:8993/solr/blackligh"
```

Comment:
```suggestion
                      ENV["SOLR_URL"] || "http://localhost:8993/solr/blacklight"
```

Reply:
oh, that shouldn't even be there... just added to my local for testing purposes 🤦‍♂